qa/pom.xml

```diff
@@ -152,6 +152,11 @@
     <profiles>
         <profile>
             <id>vagrant</id>
+            <activation>
+                <property>
+                    <name>vagrant</name>
```

Comment:
Robert [proposed](https://github.com/elastic/elasticsearch/pull/12646#issuecomment-129041593) to use `tests.vagrant` which is more compliant to other properties we have.